packages/scvelo/meta.yaml

```diff
@@ -0,0 +1,17 @@
+name: scvelo
```

Comment:
Is this just an ID? Otherwise, I'd change it to _scVelo_ as in the description.

Reply:
I think so, let's find out.

```suggestion
name: scVelo
```